src/System.Private.CoreLib/src/System/StartupHookProvider.cs

```diff
@@ -15,6 +15,7 @@ namespace System
 {
     internal class StartupHookProvider
     {
+        private static string startupHookTypeName = "StartupHook";
```

Comment:
`static const` isn't a thing, but `static readonly` is and is probably what you want.